styles.css

```diff
@@ -0,0 +1,77 @@
+body {
+  margin: -1px;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+header {
+  display: flex;
+  background: #3c3a39;
+  color: #fff;
+  justify-content: space-around;
+  height: 48px;
+  align-items: center;
+  font-size: 18px;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  background: #1c1a19;
+  color: #fff;
+  height: 100vh;
+  justify-content: center;
+  width: 100%;
+  padding: 24px;
+}
+
+.intro-text {
+  display: flex;
+  flex-direction: column;
+  height: 300px;
+  justify-content: space-between;
+}
```

Comment:
- [ ] Kindly use the `Crete Round` font-family for the **_'Hey There, I'm Carlos'_** text and `Roboto` for the sub-text and description. These were the font families used for the Headline section of your design.

- [ ] Kindly use a `margin-top` that is similar to that of the Figma's design;
### Your Design
![image](https://user-images.githubusercontent.com/77106962/143398022-81abab49-b590-407f-98e8-a72f4a5e1081.png)

### Figma's Design
![image](https://user-images.githubusercontent.com/77106962/143398237-53e235b7-58a6-4f85-a941-0874c2262528.png)
In the snapshots above, kindly notice the difference in the spacing. It would be nice to fix this because it would make your portfolio look attractive to recruiters.

Reply:
Thanks, same thing here though, the original font from Figma is not accepted by linters, I'm not sure if I did something wrong but I wanted to let you know @Gambit142